packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -1260,6 +1320,11 @@ export enum InstanceSize {
    */
   XLARGE56 = '56xlarge',
 
+  /**
+   * Instance size XLARGE56 (96xlarge)
```

Comment:
s/XLARGE56/XLARGE96

Reply:
Good catch, fixed

Reply:
may have forgotten to push the change?

Reply:
Didn't stage them 🤦 Should be good now, sorry